pom.xml

```diff
@@ -95,6 +96,12 @@
       <version>9.3</version>
     </dependency>
 
+    <dependency>
```

Comment:
Ick, no. Can we please stop adding third party logging libraries willy nilly. java.util.logging should be all we need.

Reply:
This library is used in Maven plugins ... so does Maven support `java.util.logging`?

For me discusion is still open - we can try one again find gold way for logging on dev list.

Reply:
Third party logging frameworks haven't been needed since Java 1.4. Post log4shell I have no remaining patience for external dependencies that simply duplicate what's already in the JDK. The specific printlns being replaced here can be handled by java.util.logging, and any plugins using other frameworks should be corrected.

Reply:
@elharo done. I removed slf4j and Java Core Logger is in use now.

Reply:
@elharo @Koziolek 
My question is still open how java logging will be supported by Maven?